packages/optimizer/package.json

```diff
@@ -42,7 +42,7 @@
     "htmlparser2": "4.1.0",
     "https-proxy-agent": "5.0.0",
     "lru-cache": "6.0.0",
-    "node-fetch": "2.6.0",
+    "node-fetch": "2.6.1",
```

Comment:
Could this be `^2.6.1` instead? Really, all of these dependencies ought to be semver ranges so that consumers don't have to wait for package bumps in this project to receive bug fixes and security patches.

Reply:
Sure will change!